src/ocean/Ocean.H

```diff
@@ -158,8 +153,9 @@ protected:
 
 public:
     //! constructor
-    Ocean(Teuchos::RCP<Epetra_Comm> Comm,
-          ParameterList oceanParamList);
+    Ocean(Teuchos::RCP<Epetra_Comm> Comm);
+    Ocean(Teuchos::RCP<Epetra_Comm> Comm, Model::ParameterList oceanParamList);
+    Ocean(Teuchos::RCP<Epetra_Comm> Comm, const Teuchos::ParameterList& oceanParamList);
```

Comment:
The implementation only has 2 constructors.

Reply:
ok2